src/Coverage.jl

```diff
@@ -232,27 +246,80 @@ module Coverage
     child folders. Default folder is "src", which is useful for the primary case
     where Coverage is called from the root directory of a package.
     """
-    function process_folder(folder="src")
+    function process_folder(folder="src", exclusions_file=".coverage.yml")
```

Comment:
What is the motivation for making the name of the YAML filed configurable. What are situations where you would expect that you'd use this?